### PR TITLE
Unbreak on OpenBSD

### PR DESCRIPTION
**Pull request by buzzdeee**
Create /etc/environment file before trying to manipulate it
On OpenBSD such a file doesn't exist

The change that introduced that fiddling with the /etc/environment file broke the module on OpenBSD,
since the file doesn't exist in the first place, and file_line then errors out.
As a quick fix, just ensure the file exists, before trying to edit it.

### CODE REVIEW

**Comment by buzzdeee** [comment]
Hi,

something like a central /etc/environment doesn't exist on OpenBSD, as well as all those /etc/alternatives and friends. But getting back to java, with regard to java, on OpenBSD there is the javaPathHelper package, see:
http://cvsweb.openbsd.org/cgi-bin/cvsweb/ports/java/javaPathHelper/

Package specs define a minimum java version that has to be installed, if that is met, i.e. 
MODJAVA_VER=    1.7+

When the package finds jdk-1.7 or higher installed, then dependencies are met.
The javaPathHelper utility that scans installed packages dependencies to jdk versions, and based on that, it gives the full path to a java binary or java home like this:

> javaPathHelper -c puppetdb
/usr/local/jdk-1.8.0/bin/java
> javaPathHelper -h puppetdb
/usr/local/jdk-1.8.0

so different packages might require installations of different java/jdk versions, and all works kind of flawlessly with each other.

So, the javaPathHelper is fine for softwares that are installed via packages, but doesn't help for randomly installed jars.

In short, fiddling with that file at all could also be omitted on OpenBSD.

**Comment by tphoney** [comment]
Hi @buzzdeee , thanks for the work you have put into this pull request. However due to inactivity and failing unit tests and needing rebased this pull request is being closed. If you want to re-submit this PR with the necessary changes, we will gladly look at the new pull request.
